src/parser.rs

```diff
@@ -119,7 +119,7 @@ pub fn parse_lcov<T: Read>(
     loop {
         l.clear();
 
-        let num_bytes = try!(lcov_reader.read_until(b'\n', &mut l));
+        let num_bytes = r#try!(lcov_reader.read_until(b'\n', &mut l));
```

Comment:
Could you replace usage of the try macro (deprecated) with the `?` operator?

Reply:
Yes, I have updated the PR.